open-models/serving/vertex_ai_text_generation_inference_gemma.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<p><a href="https://github.com/huggingface/Google-Cloud-Containers" target="_blank"><strong>Hugging Face DLCs</strong></a> are pre-built and optimized Deep Learning Containers (DLCs) maintained by Hugging Face and Google Cloud teams to simplify environment configuration for your ML workloads.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>